Doc/library/dbm.rst

```diff
@@ -137,37 +137,39 @@ then prints out the contents of the database::
 The individual submodules are described in the following sections.
 
 
-:mod:`dbm.gnu` --- GNU's reinterpretation of dbm
-------------------------------------------------
+:mod:`dbm.gnu` --- GNU database manager
+---------------------------------------
 
 .. module:: dbm.gnu
    :platform: Unix
-   :synopsis: GNU's reinterpretation of dbm.
+   :synopsis: GNU database manager
 
 **Source code:** :source:`Lib/dbm/gnu.py`
 
 --------------
 
-This module is quite similar to the :mod:`dbm` module, but uses the GNU library
-``gdbm`` instead to provide some additional functionality.  Please note that the
-file formats created by :mod:`dbm.gnu` and :mod:`dbm.ndbm` are incompatible.
+The :mod:`dbm.gnu` module provides an interface to the :abbr:`GDBM (GNU dbm)`
+library, similar to the :mod:`dbm.ndbm` module, but with additional
+functionality like crash tolerance.
 
-The :mod:`dbm.gnu` module provides an interface to the GNU DBM library.
-``dbm.gnu.gdbm`` objects behave like mappings (dictionaries), except that keys and
-values are always converted to bytes before storing.  Printing a ``gdbm``
-object doesn't print the
-keys and values, and the :meth:`items` and :meth:`values` methods are not
-supported.
+:class:`!gdbm` objects behave like :term:`mappings <mapping>`,
+except that keys and values are always converted to :class:`bytes` before storing.
+Printing a :class:`!gdbm` object doesn't print the keys and values,
+and the :meth:`!items` and :meth:`!values` methods are not supported.
```

Comment:
Reduced duplication (consolidate the two first sentences in the two paragraphs), use a glossary link instead of parenthesised hint, and mark up stuff properly.

Reply:
Not sure I like this paragraph too much.  It starts by saying that they behave like mappings, and then mentions a single exception.  Then it adds two more exceptions, saying what it doesn't print rather than what it prints.

I'd list list all the exceptions together, and mention what it prints (or maybe show it in an example somewhere).

Reply:
I'm not sure if the "print" subsentence is needed. It seems strange to me to even mention it.

Reply:
How about this:

```
:class:`!gdbm` objects behave like :term:`mappings <mapping>`,
except that keys and values are always converted to :class:`bytes` before storing,
and the :meth:`!items` and :meth:`!values` methods are not supported.
```

Reply:
That's definitely better!  Perhaps we could use "similarly to" instead of "like" (to me "like" has stronger implications).

Regarding the printed output, how do the `str`/`repr` look like?  If it's something worth mentioning, you could add a sentence like `In addition, printing a :class:`!gdbm` shows a nicely formatted table.` (or whatever).

Reply:
Yes, "similar to" is better than "like"; good suggestion! Regarding printed output, I'm not sure it is worth mentioning at all.

Reply:
Applied!